dist/file-preview-card-group/file-preview-card-group.css

```diff
@@ -16,6 +16,10 @@ div.file-preview-card-group ul li {
     height: max-content;
 }
 
+.file-preview-card-group--card-description {
```

Comment:
The proper BEM for this would be `file-preview-card-group__card-description`, but `file-preview-card-group` is supposed to be just a responsive layout assembler. Also, since descriptions are item-specific, this doesn't look to be the right place to add description.

I think we should add the description in `file-preview-card` as `file-preview-card__description`.

Reply:
I agree and I wanted to do it that way, but the problem is that this is a _sibling_ of `file-preview-card` and not a child, so changing it would require breaking changes to markup.

Reply:
Yikes! We should have caught this. We shouldn't have inline styles here or inside. As far as structure, I think that's the issue - it shouldn't be a _sibling_. So we don't seem to have a class used here though, which means nothing would break, right? Maybe I'm missing something.

<kbd><img width="1418" alt="image" src="https://github.com/user-attachments/assets/1387fd48-44af-40bd-88c8-293619c9d514"></kbd>

I hesitate to keep moving in the wrong direction because we've made the wrong turn.

Reply:
We don't have a class there, but I was discussing with @agliga offline about how there _should_ be since otherwise there will be a bunch of inline styles in eBayUI. I'd like for this to be called `file-preview-card__description`, but since it's a sibling that would be a violation of BEM. I don't think it makes sense to add an extra level of nesting to `file-preview-card`, but maybe you think differently?